public/assets/css/sign-in.css

```diff
@@ -6,10 +6,36 @@
   clear: both;
 }
 .sign-in-bttn {
-  float: right;
 }
```

Comment:
Since there's no more styles here, could make sense to remove it altogether.

```suggestion
```

Reply:
Good point! thx